akka-server-app/src/main/scala/de/mukis/AkkaServerApp.scala

```diff
@@ -0,0 +1,11 @@
+package de.mukis
+
+import akka.actor.{ ActorSystem, Props }
+import scala.concurrent.duration._
+import com.typesafe.config.ConfigFactory
+
+object AkkaServerApp extends App{
+  val config = ConfigFactory.load
```

Comment:
One nice thing would be to show that the `application.ini` is loaded. E.g. give the application a single parameter
`AkkaServerApp`.

```scala
val systemName = args(0)
ActorSystem(systemName,...)
```